docs/guide/database-insert-update.md

```diff
@@ -16,21 +16,27 @@ When we have an existing piece document that to update, we use the `update()` me
 | `piece` | TRUE | The document object that will *replace* the existing database document. |
 | `options` | FALSE | An options object, currently only used for internal draft state management. |
 
-For example, if we had a dog adoption website we may want to update `dog` pieces to indicate whether someone requested information about them. This could be after a contact form receives a submission about the dog.
+For example, if we had a dog adoption website that used an external API, we may want to update `dog` pieces periodically with adoption status. This may be using a command line task that is run regularly by a cron job.
```

Comment:
OK, so `req` is going to be a task req with admin permissions in your example, but I feel like we're just asking for a lot of frustrated devs if we don't mention the `{ permissions: false }` option is the right way to go if you *do* want to let an anonymous site visitor insert or update a piece.

Reply:
I updated the paragraph about this to try to highlight it a bit more.